atom/common/native_mate_converters/blink_converter.cc

```diff
@@ -238,7 +238,6 @@ bool Converter<blink::WebMouseWheelEvent>::FromV8(
   dict.Get("accelerationRatioX", &out->accelerationRatioX);
   dict.Get("accelerationRatioY", &out->accelerationRatioY);
   dict.Get("hasPreciseScrollingDeltas", &out->hasPreciseScrollingDeltas);
-  dict.Get("canScroll", &out->canScroll);
```

Comment:
If we want to maintain backwards compatibility, we could use the method that replaced that member https://codereview.chromium.org/2052663004/patch/80001/90009